Sources/PaymentsCore/Networking/APIRequestBody.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+public protocol APIRequestBody: Encodable {
+    func encoded() -> Data?
+}
+
+extension APIRequestBody {
+
+    public func encoded() -> Data? {
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        return try? encoder.encode(self)
```

Comment:
We can keep encoding DRY by providing a public extension in the core module to produce a `Data` encoded `APIRequestBody`.